src/lib/backend/services/addressService.ts

```
@@ -0,0 +1,80 @@
+
+  // 주소 삭제
+  static async deleteAddress(addressId: number): Promise<void> {
+    const { error } = await client.DELETE("/api/addresses/{addressId}", {
+      params: { path: { addressId } },
+    });
+
+    if (error) {
+      throw new Error("주소 삭제에 실패했습니다.");
+    }
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)